source/unified-test-format/schema-1.3.json

```diff
@@ -194,14 +210,47 @@
       "required": ["client", "events"],
       "properties": {
         "client": { "type": "string" },
-        "events": {
-          "type": "array",
-          "items": { "$ref": "#/definitions/expectedEvent" }
+        "eventType": {
```

Comment:
New `invalid` tests:

 * `expectedEventsForClient-eventType-enum.yml` with invalid string value
 * `expectedEventsForClient-eventType-type.yml` with invalid type

Reply:
Related to my comment about `ignoreResultAndError`, each file in `invalid` should have a single JSON schema failure.